src/javascript/JContent/EditFrame/EditFrame.scss

```diff
@@ -175,14 +175,15 @@
 }
 
 :global(.noprefix) .marginTop {
-    margin-top: 36px;
+    // FIXME override margin-top for empty area in iframe without !important
+    margin-top: 46px !important;
```

Comment:
Still outstanding; should be fixed with https://jira.jahia.org/browse/BACKLOG-23410